new sensors

rtravis-whoi opened the pull request:
Added new sensors.  See attached for the additions in red font & also refer to WHOI_Asset_Tracking Ss for verification.

[sensor_bulk_load-AssetRecord_2024-01-29.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14090183/sensor_bulk_load-AssetRecord_2024-01-29.xlsx)

[WHOI_Asset_Tracking_2024-01-29.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14090865/WHOI_Asset_Tracking_2024-01-29.xlsx)

snwhite commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -1302,12 +1320,19 @@ CGINS-FDCHPA-70601,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER W
 CGINS-FDCHPB-12802,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,212802,,20210901,,,Woods Hole Oceanographic Institution
 CGINS-FDCHPB-12803,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,212803,,20210901,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-34110,A04359,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,234110,,20231109,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-34111,A04360,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,234111,,20231109,,,Woods Hole Oceanographic Institution
```
Looks good. 
Other new units are OSU units.

snwhite commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -2400,6 +2428,27 @@ CGINS-PRESFC-01398,N00683,Sensor,0,Seafloor Pressure: PRESF Series C,PRESF SERIE
+CGINS-PRTSZA-02268,A04369,Sensor,0,Particle Size Analyzer: PRTSZ Series A,PRTSZ SERIES A,Sequoia,LISST-200X,2268,,20231024,,,Woods Hole Oceanographic Institution
+CGINS-PRTSZA-02269,A04370,Sensor,0,Particle Size Analyzer: PRTSZ Series A,PRTSZ SERIES A,Sequoia,LISST-200X,2269,,20231024,,,Woods Hole Oceanographic Institution
+CGINS-PRTSZA-02270,A04371,Sensor,0,Particle Size Analyzer: PRTSZ Series A,PRTSZ SERIES A,Sequoia,LISST-200X,2270,,20231024,,,Woods Hole Oceanographic Institution
+CGINS-PRTSZA-02271,A04372,Sensor,0,Particle Size Analyzer: PRTSZ Series A,PRTSZ SERIES A,Sequoia,LISST-200X,2271,,20231024,,,Woods Hole Oceanographic Institution
```
Looks good.
Cost is $47,000 each.

cooleyky replied:
PRESFs and PRTSZs look good

snwhite reviewed (approved):
Approved but provide some costs if those want/need to be added in.

cooleyky commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -2380,6 +2405,9 @@ CGINS-PHSENF-P0180,A01713,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Su
 CGINS-PHSENF-P0183,A01716,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Sunburst Sensors,SAMI-pH,P0183,,20150818,18827.5,,
+CGINS-PLIMSA-00199,A04366,Sensor,0,Phytoplankon Imagery: PLIMS Series A,PLIMS SERIES A,McLane,ImagingFlowCytoBot (IFCB),199,,20231013,,,Woods Hole Oceanographic Institution
+CGINS-PLIMSA-00200,A04365,Sensor,0,Phytoplankon Imagery: PLIMS Series A,PLIMS SERIES A,McLane,ImagingFlowCytoBot (IFCB),200,,20231013,,,Woods Hole Oceanographic Institution
+CGINS-PLIMSA-00206,A04386,Sensor,0,Phytoplankon Imagery: PLIMS Series A,PLIMS SERIES A,McLane,ImagingFlowCytoBot (IFCB),206,,20240102,,,Woods Hole Oceanographic Institution
```
PLIMS look good

cooleyky commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -2589,6 +2638,8 @@ CGINS-WAVSSA-06311,A01671,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SE
 CGINS-WAVSSA-06421,,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SERIES A (TRIAXYS NW),AXYS Technologies,TRIAXYS Directional Wave Sensor,TAS06421,,20150715,24039.99,,Woods Hole Oceanographic Institution
 CGINS-WAVSSA-06431,A01626,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SERIES A (TRIAXYS NW),AXYS Technologies,TRIAXYS Directional Wave Sensor,TAS06431,,20150715,24039.99,Lost at sea on GI01SUMO-00004,Woods Hole Oceanographic Institution
+CGINS-ZPLSCC-55070,A01831,Sensor,0,Bio-acoustic Sonar: ZPLSC Series C,ZPLSC SERIES C (AZFP),ASL Environmental,AZFP,55070,,,,converted from ZPLSG-A toZPLSC-C for Pioneer MAB,Woods Hole Oceanographic Institution
+CGINS-ZPLSCC-55071,A01832,Sensor,0,Bio-acoustic Sonar: ZPLSC Series C,ZPLSC SERIES C (AZFP),ASL Environmental,AZFP,55071,,,,converted from ZPLSG-A toZPLSC-C for Pioneer MAB,Woods Hole Oceanographic Institution
```
ZPLSCs look good

cooleyky reviewed (approved):
looks good

rtravis-whoi commented:
I added in the following costs:
ADCPTQ - 34160
ADCPUP - 20959
PLIMSA - 135375
PRESFD - 23913
PRESFE - 23195
PRTSZA - 47000
Also added FDCHPB controller 16 - it isn't in asset tracking yet as upgraded, but will be the A converted to B
Also removed reference to 'for MAB' in comments for converted ZPLSCCs

snwhite reviewed (approved):
Looks good to me!

cooleyky commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -2638,8 +2639,8 @@ CGINS-WAVSSA-06311,A01671,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SE
-CGINS-ZPLSCC-55070,A01831,Sensor,0,Bio-acoustic Sonar: ZPLSC Series C,ZPLSC SERIES C (AZFP),ASL Environmental,AZFP,55070,,,,converted from ZPLSG-A toZPLSC-C for Pioneer MAB,Woods Hole Oceanographic Institution
-CGINS-ZPLSCC-55071,A01832,Sensor,0,Bio-acoustic Sonar: ZPLSC Series C,ZPLSC SERIES C (AZFP),ASL Environmental,AZFP,55071,,,,converted from ZPLSG-A toZPLSC-C for Pioneer MAB,Woods Hole Oceanographic Institution
+CGINS-ZPLSCC-55070,A01831,Sensor,0,Bio-acoustic Sonar: ZPLSC Series C,ZPLSC SERIES C (AZFP),ASL Environmental,AZFP,55070,,,,converted from ZPLSG-A to ZPLSC-C,Woods Hole Oceanographic Institution
+CGINS-ZPLSCC-55071,A01832,Sensor,0,Bio-acoustic Sonar: ZPLSC Series C,ZPLSC SERIES C (AZFP),ASL Environmental,AZFP,55071,,,,converted from ZPLSG-A to ZPLSC-C,Woods Hole Oceanographic Institution
```
looks good!

cooleyky reviewed (approved):
updates all look good!

rtravis-whoi commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Pioneer 20 AM Updates](https://trello.com/c/22LKGqi6/232-pioneer-20-am-updates)
